add-on/panels/js/slideshow.jsx

```diff
@@ -0,0 +1,281 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var loop = loop || {};
+loop.slideshow = (function(_, mozL10n) {
+  "use strict";
+
+// App state
```

Comment:
Copied the used code but could not separate cleanly.  Using the original slideshow.jsx running code the vendor version is just for license.  The slideshow had scope issues from the separate function closures used.